### PR TITLE
Managing Windows Servers Remotely with Powershell Blog Post

### PR DESCRIPTION
**Pull request by gomker**
Hello,

My name is Derek Lane, I work on the Critical Application Services team as a Application Services Engineer and  have been working at Rackspace 10+ years.
Our team focuses on Application Support for Rackspace and my area of focus is in .NET application support and PowerShell

I accept  redaction, editing or structure changes by technical writer staff
Desired date of publication - as soon as possible.
I will accept comments.

Regards,

Derek Lane

### CODE REVIEW

File: _posts/2016-03-10-Managing-Windows-Servers-Remotely-with-PowerShell.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+author: Derek Lane
+published: true
+categories:
+    - devops
+    - automation
+    - powershell
+---
+#Managing Windows Servers Remotely with PowerShell
+
+Automation in Windows has historically been a challenge due to lack of built in tools for remote management.  In the past few years the enhancements to PowerShell and WinRM (Windows Remote Management) have forged a path that is now more on par with other OS's in regards to remote access.
```
Comment (catlook) [inline _posts/2016-03-10-Managing-Windows-Servers-Remotely-with-PowerShell.md]: ... few years, the enhancements ...
... with other operating systems with regards ...

File: _posts/2016-03-10-Managing-Windows-Servers-Remotely-with-PowerShell.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+<!-- more -->
+
+One of the primary benefits in using PowerShell is that there is no need to install a third-party agent, WinRM is usually configured by most Cloud Providers (Rackspace enables and HTTPS listener during provisioning)
```
Comment (catlook) [inline _posts/2016-03-10-Managing-Windows-Servers-Remotely-with-PowerShell.md]: ... benefits of using  ...
... a third-party agent. WinRM is configured by most  Cloud Providers, and Rackspace enables an HTTPS listener during provisioning.

File: _posts/2016-03-10-Managing-Windows-Servers-Remotely-with-PowerShell.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+###What do you need
+
+[PoshStack](https://developer.rackspace.com/blog/Introducing-PoshStack-the-PowerShell-client-for-OpenStack/) is an excellent set of PowerShell tools will allow us to automate gathering the information we need to connect to our servers.
```
Comment (catlook) [inline _posts/2016-03-10-Managing-Windows-Servers-Remotely-with-PowerShell.md]: ... PowerShell tools that enables us to automate the gathering of the information that we ...

File: _posts/2016-03-10-Managing-Windows-Servers-Remotely-with-PowerShell.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+Built on OpenStack .NET SDK , we can leverage these functions in PowerShell for many other management tasks as well.
+
+** Example listing servers by name match **
+	
+	Get-OpenStackComputeServer -Account ORD | Where-Object {$_.name -like 'ServerName*'}
+
+###Challenges in Authentication
+Microsoft usually assumes you are in an Active Directory Domain utilizing credentials that can access all servers in your environment.  This may be true in certain cases, however when utilizing the cloud it is not a certainty.  In this example I am using a set of credentials that works across all of my devices to keep the scope of this article simple
```
Comment (catlook) [inline _posts/2016-03-10-Managing-Windows-Servers-Remotely-with-PowerShell.md]: ... however, when utilizing the cloud, it  ...
... this example, I use a set ...
... this article simple.

File: _posts/2016-03-10-Managing-Windows-Servers-Remotely-with-PowerShell.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+    $ConnectedServers = Get-PSSession
+ 
+**Executing a command to all the servers at once**
+
+	Invoke-Command -Session $ConnectedServers -ScriptBlock {Get-Service RackspaceCloudServersAgent}
+
+
+This is just a quick primer on how to use PowerShell and remote sessions. WinRM is the foundation for automation in Windows for DSC(Desired State Configuration) or Ansible among others.
+
+ Look for more to come in how to leverage these concepts further using DSC
```
Comment (catlook) [inline _posts/2016-03-10-Managing-Windows-Servers-Remotely-with-PowerShell.md]: Look for future articles describing how to leverage these concepts further using DSC.

File: _posts/2016-03-10-Managing-Windows-Servers-Remotely-with-PowerShell.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+![Example]({% asset_path 2016-03-10-Managing-Windows-Servers-Remotely-with-PowerShell/PoshCode.png %})
+ 
+###Further reading
+
+[PoshStack](https://github.com/rackerlabs/PoshStack)
+
+[PowerShell Sessions](https://technet.microsoft.com/en-us/library/hh849717.aspx)
+
+[DSC - Desired State Configuration](https://blogs.technet.microsoft.com/privatecloud/2013/08/30/introducing-powershell-desired-state-configuration-dsc/)
```
Comment (catlook) [inline _posts/2016-03-10-Managing-Windows-Servers-Remotely-with-PowerShell.md]: [ ed note: Thanks for sharing!  Use any edits that makes sense and ask me any questions!  -- Cat ]

**Comment by smashwilson** [comment]
:shipit:

**Pull request merged**